pkg/test/expectations/expectations.go

```diff
@@ -137,3 +145,37 @@ func ExpectReconcileSucceeded(reconciler reconcile.Reconciler, key client.Object
 	_, err := reconciler.Reconcile(context.Background(), reconcile.Request{NamespacedName: key})
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func ExpectPodsEvictingSucceeded(c client.Client, pods ...*v1.Pod) {
```

Comment:
Thoughts on calling this 
```ExpectEvicted(c client.Client, pods ...*v1.Pod)```
and then actually doing the deletion after we verify the deletion timestamp is set?

Reply:
I would rather have this check just check it, then separately delete the pod, just in case there is a time where we don't immediately want to delete it, and check some other things first.